src/plugins/path/path.c

```diff
@@ -12,6 +12,60 @@
 #include "kdbconfig.h"
 #endif
 
+static int validateKey(Key *key, Key *parentKey)
```

Comment:
Seems like code is not formatted yet. And parentKey should be renamed to errorKey?

Don't you want to export this function? (Why is it static?)